include/flags.hpp

```diff
@@ -861,6 +861,7 @@
 #define X_LOOK_ONLY                 94      // Can only look through this exit (free scout)
 #define X_LOCK_AFTER_USAGE          95      // Exit re-locks after single usage
 #define X_NO_KNOCK_SPELL            96      // Exit immune to knock spell
-#define MAX_EXIT_FLAGS              97      // Incriment when adding... check structs.h for max
+#define X_SEL_KENKU                 97      // race selective: kenku
```

Comment:
is there nothing closer to the other ones? I recall it being some math to figure out which flag it was.

Reply:
It checks the flags individually. No math.